packages/sil-governance/src/routes/create-proposal/components/CommitteeSelect.tsx

```diff
@@ -0,0 +1,68 @@
+import { ElectionRule } from "@iov/bns";
+import { FormApi } from "final-form";
+import { Block, SelectFieldForm, Typography } from "medulas-react-components";
+import React, { Dispatch, SetStateAction, useState } from "react";
+
+export const COMMITTEE_FIELD = "Committee";
+const COMMITTEE_FIELD_INITIAL = "Select a committee";
+
+interface Props {
+  form: FormApi;
+  electionRules: ElectionRule[];
+  changeElectionRuleId: Dispatch<SetStateAction<number>>;
+}
+
+const CommitteeSelect = ({ form, electionRules, changeElectionRuleId }: Props): JSX.Element => {
+  const committeeItems = electionRules.map(rule => {
+    return {
+      name: rule.title,
+    };
+  });
+
+  const [quorum, setQuorum] = useState();
+  const [threshold, setThreshold] = useState();
+  const [period, setPeriod] = useState();
+
+  const changeCommittee = (committeeName: string): void => {
+    const committee = electionRules.find(rule => rule.title === committeeName);
```

Comment:
Is there a more solid way for this check by storing the ID in the list or something like that? It would be allowed (even if not useful) to have two committees that use the same name